src/mediapipe_internal/mediapipegraphdefinition.hpp

```diff
@@ -82,6 +64,17 @@ class MediapipeGraphDefinition {
 
     Status create(std::shared_ptr<MediapipeGraphExecutor>& pipeline, const KFSRequest* request, KFSResponse* response);
 
+    static std::string getStreamName(const std::string& streamFullName) {
```

Comment:
Move implementation & include to cpp

Reply:
It has a visibility warning as error then.